.github/workflows/PR.yml

```diff
@@ -14,7 +14,7 @@ concurrency:
 
 jobs:
   run_ci:
-    name: Run CI
+    name: CI
```

Comment:
I thought you said you were going to drop the renames and just aggregate the checks?

Reply:
I still see a lot of renames that are unrelated to the PR description.